src/colormanager.cpp

```diff
@@ -9,13 +9,49 @@
 
 using namespace podboat;
 
-namespace newsboat {
-
-ColorManager::ColorManager()
-{
+namespace {
+const std::map<std::string, newsboat::TextStyle> default_styles {
+	{ "listnormal",           {"default", "default", {}} },
+	{ "listfocus",            {"yellow",  "blue",    {"bold"}} },
+	{ "listnormal_unread",    {"default", "default", {"bold"}} },
+	{ "listfocus_unread",     {"yellow",  "blue",    {"bold"}} },
+	{ "info",                 {"yellow",  "blue",    {"bold"}} },
+	{ "background",           {"white",   "black",   {}} },
+	{ "article",              {"white",   "black",   {}} },
+	{ "end-of-text-marker",   {"blue",    "black",   {"bold"}} },
```

Comment:
As far as I can see, none of the stfl files mentioned that the background is black. Shouldn't we use `default` here?

```suggestion
	{ "end-of-text-marker",   {"blue",    "default",   {"bold"}} },
```

Reply:
Good catch, will fix

Reply:
Done